packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -10,6 +10,11 @@ import styles from './styles/dropdown.module.scss';
 
 export type DropdownRootProps = {
     children?: ReactNode;
+    /**
+     * When set to true, interaction with outside elements will be disabled and only menu content will be visible to screen readers.
+     * @default false
```

Comment:
Why is it set to `false`? Default is `true` in Radix, why would you need to allow interaction with the outside elements?